extract_endpoint/src/extract_endpoint/post_to_endpoint.py

```diff
@@ -31,10 +36,32 @@ def post_stream(stream: typing.IO[bytes],
                          data={'salt': salt, 'signature': signature, 'timestamp': timestamp})
 
 
+def send_to_tl_app(url: str, data: typing.Dict[str, str]) -> int:
+    return requests.post(url, data=data).status_code
```

Comment:
You wrap your other posts in `try/except` blocks, but not this one. Intentional?

Reply:
no - should do this one too. good catch.